prov/psm/src/psmx.h

```diff
@@ -614,12 +614,6 @@ int	psmx_domain_check_features(struct psmx_fid_domain *domain, int ep_cap);
 int	psmx_domain_enable_ep(struct psmx_fid_domain *domain, struct psmx_fid_ep *ep);
 void	psmx_domain_disable_ep(struct psmx_fid_domain *domain, struct psmx_fid_ep *ep);
 
-void	psmx_ns_start_server(struct psmx_fid_fabric *fabric);
-void	psmx_ns_stop_server(struct psmx_fid_fabric *fabric);
-void	psmx_ns_add_local_name(int service, psm_epid_t name);
-void	psmx_ns_del_local_name(int service, psm_epid_t name);
-void	*psmx_ns_resolve_name(const char *server, int *service);
```

Comment:
ok - here's where service as int is coming from.  Let's go ahead and separate out the existing ns and worry about generalization (see my comment above) later.